src/modules/OpenChannel/components/OpenChannelMessage/utils.ts

```diff
@@ -17,9 +17,17 @@ export const SendingMessageStatus = {
   PENDING: 'pending',
 };
 
-export const getMessageType = (message: UserMessage | FileMessage | AdminMessage): string => {
+type MessageTypeOptions = {
+  isOgMessageEnabledInOpenChannel?: boolean;
+};
+
+export const getMessageType = (
+  message: UserMessage | FileMessage | AdminMessage,
+  options?: MessageTypeOptions,
+): string => {
+  const isOgMessageEnabledInOpenChannel = options?.isOgMessageEnabledInOpenChannel;
```

Comment:
cc @AhyoungRyu , its better to fallback to text message when og message is not present